test/rails_app/app/views/users/invitations/new.html.erb

```diff
@@ -1,15 +1,21 @@
 <h2>Send an invitation</h2>
 
-<%= form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => { :method => :post } do |f| %>
+<%= form_for(resource, as: resource_name, url: invitation_path(resource_name), html: { method: :post }) do |f| %>
   <%= devise_error_messages! %>
```

Comment:
I think it should be changed to render partial

Reply:
Done! See latest commit (https://github.com/scambra/devise_invitable/pull/786/commits/568ea18b66e879fe3c8ef72ab04ce6981833f61d)